bench/wtperf/wtperf.c

```diff
@@ -1307,23 +1416,27 @@ monitor(void *arg)
                 first = false;
             }
             (void)fprintf(jfp, "\"localTime\":\"%s\",\"wtperf\":{", buf);
-            /* Note does not have initial comma before "read" */
-            (void)fprintf(jfp, "\"read\":{\"ops per sec\":%" PRIu64 ",\"average latency\":%" PRIu32
-                               ",\"min latency\":%" PRIu32 ",\"max latency\":%" PRIu32 "}",
-              cur_reads, read_avg, read_min, read_max);
+            /* Note does not have initial comma before "insert" */
             (void)fprintf(jfp,
-              ",\"insert\":{\"ops per sec\":%" PRIu64 ",\"average latency\":%" PRIu32
+              "\"insert\":{\"ops per sec\":%" PRIu64 ",\"average latency\":%" PRIu32
               ",\"min latency\":%" PRIu32 ",\"max latency\":%" PRIu32 "}",
               cur_inserts, insert_avg, insert_min, insert_max);
+            (void)fprintf(jfp,
+              ",\"modify\":{\"ops per sec\":%" PRIu64 ",\"average latency\":%" PRIu32
+              ",\"min latency\":%" PRIu32 ",\"max latency\":%" PRIu32 "}",
+              cur_modifies, modify_avg, modify_min, modify_max);
+            (void)fprintf(jfp, ",\"read\":{\"ops per sec\":%" PRIu64 ",\"average latency\":%" PRIu32
+                               ",\"min latency\":%" PRIu32 ",\"max latency\":%" PRIu32 "}",
+              cur_reads, read_avg, read_min, read_max);
             (void)fprintf(jfp,
               ",\"update\":{\"ops per sec\":%" PRIu64 ",\"average latency\":%" PRIu32
               ",\"min latency\":%" PRIu32 ",\"max latency\":%" PRIu32 "}",
               cur_updates, update_avg, update_min, update_max);
             fprintf(jfp, "}}\n");
         }
 
-        if (latency_max != 0 &&
-          (read_max > latency_max || insert_max > latency_max || update_max > latency_max)) {
+        if (latency_max != 0 && (insert_max > latency_max || modify_max > latency_max ||
+                                  read_max > latency_max || update_max > latency_max)) {
```

Comment:
Whitespace on the continuation indent??  Sue flagged something like this earlier, but I know the style and indentation rules have recently changed.  If this file passes the style checker, then I'm good.

Reply:
This is because of new clang format, continuation indentation is applying after the nested brackets. I even tried to bring the entire second block down, still, the format is returning to the same way.